.github/workflows/molecule-tests.yml

```diff
@@ -0,0 +1,124 @@
+---
+name: Molecule tests
+
+on:
+  push:
+  pull_request:
+
+env:
+  CARTRIDGE_CLI_VERSION: '2.4.0'
+
+jobs:
+  molecule-tests-ce:
+    if: |
+      github.event_name == 'push' ||
+      github.event_name == 'pull_request' && github.event.pull_request.head.repo.owner.login != 'tarantool'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        tarantool-version: ["1.10", "2.5"]
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@master
```

Comment:
It will be more stable to bind to a specific version. I suppose, `v2` would be enough